segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -280,6 +281,15 @@ public void operationsFailed(Collection<CompletableOperation> operations) {
                 GLOBAL_OPERATION_LATENCY.reportFailValue(millis);
             }
         }
+
```

Comment:
Is this change necessary?

Reply:
Well, what I see in the diff is that you are moving `reportOperationLogSize()` from line 519 to line 284. My question is whether moving this method is necessary or not.